.github/workflows/main.yml

```diff
@@ -46,6 +46,8 @@ jobs:
         run: pushd rustworkx-core && cargo test && popd
       - name: rustworkx-core Docs
         run: pushd rustworkx-core && cargo doc && popd
+        env:
+          RUSTDOCFLAGS: '-D warnings'
```

Comment:
It is very weird that this isn't a command line option. But it is a Cargo limitation, there is not much we can do